docs/.eslintrc.js

```diff
@@ -9,6 +9,9 @@ module.exports = {
 	extends: [
 		'@nciocpl/eslint-config-react'
 	],
+  rules: {
+    'react/no-unescaped-entities': 0
+  },
```

Comment:
Let's not change the linter rules. Let's talk about the why here and what can be done to work around this.